dept44-starter/src/main/java/se/sundsvall/dept44/configuration/WebConfiguration.java

```diff
@@ -51,9 +52,11 @@
 public class WebConfiguration implements WebMvcConfigurer {
 
 	private final YAMLMapper yamlMapper;
+	private final Integer municipalityIdLocation;
```

Comment:
`municipalityIdPathPartIndex`?

Reply:
uriIndexLocation?

Reply:
Skip the "location", index will be sufficient.